README.md

```diff
@@ -47,11 +47,24 @@ If you have any questions or want to dig into other works we considered, feel fr
 
 ### Civics
 
-Topic description coming soon!
+A major lense through which governance of communities is understood is civics and citizenship. Even our own texts talk about "a civic layer for the web." But it's not clear that we really know what we mean by that! In order to think about communities, and more compellingly, what forms of space and collective action could be built around decentralized forms of governance, let's read about civics!
```

Comment:
@Mr0grog -- b/c of how @titaniumbones and I draft (maybe writing envs?) the rest of the quotations are all straight v. curly. I don't have strong preferences here but we're inconsistent throughout this README (though consistent as straight quotes in this section).

I think this nitpick might be better handled as a find and replace on the whole readme rather than this PR?

Reply:
Sure 👍  The main thing was `lens`, not the quotes, anyway.